package.json

```diff
@@ -121,12 +121,13 @@
     "prettier": "^2.4.1",
     "remark-cli": "^10.0.0",
     "remark-frontmatter": "^4.0.1",
-    "remark-preset-lint-node": "^3.2.0",
+    "remark-preset-lint-node": "^3.3.0",
     "stylelint": "^13.13.1",
     "stylelint-config-prettier": "^9.0.3",
     "stylelint-config-recommended-scss": "^4.3.0",
     "stylelint-config-sass-guidelines": "^8.0.0",
-    "stylelint-selector-bem-pattern": "^2.1.1"
+    "stylelint-selector-bem-pattern": "^2.1.1",
+    "unified-lint-rule": "^2.1.0"
```

Comment:
I think it's a good trick to unstick the lockfile, but I think the pinning in here should be removed again, so it goes back in sync with the remark(cli) version